extensions/PhabBugz/template/en/default/phabricator/table.html.tmpl

```diff
@@ -26,4 +26,10 @@
         <span class="phabricator-load-error-string"></span></td>
     </tr>
   </tbody>
+  <tr>
```

Comment:
This `<tr>` has to be inside `<tbody>` and probably hidden when there are no abandoned revisions? I honestly don’t know how to test this locally 😅

Reply:
I put the checkbox row now in a <tbody> of its own and hide it by default. I also do not show the row if there are no abandoned revisions as you suggested. If the JS and template changes check out OK, then I can put this branch on bugzilla-dev and you can test it functionally there.